src/main/java/com/ace/alfox/lib/ActionFactory.java

```diff
@@ -1,23 +1,47 @@
 package com.ace.alfox.lib;
 
 import com.ace.alfox.game.interfaces.IAction;
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.context.annotation.ClassPathScanningCandidateComponentProvider;
+import org.springframework.core.type.filter.AnnotationTypeFilter;
 
 import java.lang.reflect.InvocationTargetException;
+import java.util.HashMap;
 import java.util.Map;
 
 public class ActionFactory {
     public IAction action;
     public Map<String, Object> params;
+    /**
+     * Mappings of /perform/{action} to an IAction. Populated in the AlfoxApplication class.
+     */
+    private static Map<String, Class<IAction>> actions = new HashMap<>();
 
     public ActionFactory(IAction a, Map<String, Object> p) {
         this.action = a;
         this.params = p;
     }
 
+    public static void findActions(String namespace) throws ClassNotFoundException {
```

Comment:
I feel like `findActions` should return something. What do you think of something like `addActionsToFactory` or `addActions`?